### PR TITLE
[KYUUBI #2895] Show final info in trino engine

### PR DESCRIPTION
**Pull request by hddong**
### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [x] Add screenshots for manual tests if appropriate
trino-process

- [X] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by yaooqinn** [comment]
unnecessary and heavy for metadata operations to create operation log.

**Comment by hddong** [comment]
> unnecessary and heavy for metadata operations to create operation log.

@yaooqinn Removed from metadata oprations.

File: externals/kyuubi-trino-engine/src/main/scala/org/apache/kyuubi/engine/trino/TrinoConf.scala [hunk 1]
```diff
@@ -39,4 +39,18 @@ object TrinoConf {
       .version("1.5.0")
       .timeConf
       .createWithDefault(Duration.ofMinutes(2).toMillis)
+
+  val ENGINE_TRINO_SHOW_PROGRESS: ConfigEntry[Boolean] =
```
Comment (yaooqinn) [inline externals/kyuubi-trino-engine/src/main/scala/org/apache/kyuubi/engine/trino/TrinoConf.scala:43]: these configs does not documented

File: externals/kyuubi-trino-engine/src/main/scala/org/apache/kyuubi/engine/trino/TrinoStatusPrinter.scala [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.engine.trino
+
+import java.util.concurrent.TimeUnit._
+
+import io.airlift.units.DataSize
+import io.airlift.units.Duration
```
Comment (yaooqinn) [inline externals/kyuubi-trino-engine/src/main/scala/org/apache/kyuubi/engine/trino/TrinoStatusPrinter.scala:23]: do we need these？

File: externals/kyuubi-trino-engine/src/main/scala/org/apache/kyuubi/engine/trino/TrinoStatusPrinter.scala [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+  def printFinalInfo(
+      client: StatementClient,
+      operationLog: OperationLog,
+      debug: Boolean = false): Unit = {
+    val out = new TrinoConsoleProgressBar(operationLog)
+    val results = client.finalStatusInfo()
+    val stats = results.getStats()
+
+    val wallTime = Duration.succinctDuration(stats.getElapsedTimeMillis(), MILLISECONDS)
```
Comment (yaooqinn) [inline externals/kyuubi-trino-engine/src/main/scala/org/apache/kyuubi/engine/trino/TrinoStatusPrinter.scala:43]: why not use millis directly？

File: externals/kyuubi-trino-engine/src/main/scala/org/apache/kyuubi/engine/trino/TrinoConf.scala (hunk 1 above)
Comment (hddong) [reply externals/kyuubi-trino-engine/src/main/scala/org/apache/kyuubi/engine/trino/TrinoConf.scala]: Had move to `KyuubiConf`.

File: externals/kyuubi-trino-engine/src/main/scala/org/apache/kyuubi/engine/trino/TrinoStatusPrinter.scala (hunk 2 above)
Comment (hddong) [reply externals/kyuubi-trino-engine/src/main/scala/org/apache/kyuubi/engine/trino/TrinoStatusPrinter.scala:23]: Functions(`formatDataRate(DataSize dataSize, Duration duration, boolean longForm)`) in `FormatUtils` need these types to format output string.

File: externals/kyuubi-trino-engine/src/main/scala/org/apache/kyuubi/engine/trino/TrinoStatusPrinter.scala (hunk 3 above)
Comment (hddong) [reply externals/kyuubi-trino-engine/src/main/scala/org/apache/kyuubi/engine/trino/TrinoStatusPrinter.scala:43]: > why not use millis directly？

It is used to covert millis to Duration, to pass to functions in `FormatUtils`.

**Comment by hddong** [comment]
@yaooqinn please review this again.
cc @pan3793

File: externals/kyuubi-trino-engine/src/main/scala/org/apache/kyuubi/engine/trino/TrinoStatusPrinter.scala (hunk 3 above)
Comment (yaooqinn) [reply externals/kyuubi-trino-engine/src/main/scala/org/apache/kyuubi/engine/trino/TrinoStatusPrinter.scala:43]: what is FormatUtils？

**Comment by ulysses-you** [comment]
cc @cxzl25

File: externals/kyuubi-trino-engine/src/main/scala/org/apache/kyuubi/engine/trino/TrinoStatusPrinter.scala (hunk 3 above)
Comment (hddong) [reply externals/kyuubi-trino-engine/src/main/scala/org/apache/kyuubi/engine/trino/TrinoStatusPrinter.scala:43]: > what is FormatUtils？

https://github.com/apache/incubator-kyuubi/pull/2896/files#diff-16f6ae2d81713d0c45408cbd4a43eda8612a42e461691f04033672ad858a5e8dR34-R37
```
/** Copy from trino-cli `FormatUtils` */
public final class ProgressFormatUtils {
  private ProgressFormatUtils() {}
```
Used to format output data and process ba.

**Review by pan3793 — APPROVED** [review]

**Review by cxzl25 — APPROVED** [review]

**Comment by hddong** [comment]
thanks, merged to master

**Pull request closed without merging**